rskj-core/src/main/java/co/rsk/RskContext.java

```diff
@@ -811,17 +815,36 @@ protected SolidityCompiler buildSolidityCompiler() {
         return new SolidityCompiler(getRskSystemProperties());
     }
 
-    private TrieStore buildAbstractTrieStore(Path databasePath) {
+    @VisibleForTesting
+    protected TrieStore buildAbstractTrieStore(Path databasePath) {
```

Comment:
I'd rather you used `getTrieStore()` which is public already instead of exposing internals

Reply:
You were right, it was possible to test the `RskContext#getTrieStore` directly by mocking the _databaseDir_ property